MANIFEST.in

```
@@ -9,7 +9,7 @@ include requirements_docs.txt
 
 recursive-include figanos *.json *.mplstyle *.py *.txt
 recursive-include docs *.rst conf.py Makefile make.bat *.jpg *.png *.gif
-recursive-include data *.json *.txt
+recursive-include data *.json *.txt *.yml
```

```suggestion
recursive-include figanos/data *.json *.txt *.yml *png
```
Aha! The path is relative to the top. This line previously "worked" because txt and json are already added under "figanos"

---

Classic mistake. Good call.